src/frontend/github-oauth.ts

```diff
@@ -148,14 +151,39 @@ export default (opts: OAuthOptions): GithubOAuth => {
 		checkGithubAuth: async (req: Request, res: Response, next: NextFunction) => {
 			const traceLogsEnabled = await booleanFlag(BooleanFlags.TRACE_LOGGING, false);
 			const tracer = new Tracer(logger.child(opts), "checkGithubAuth", traceLogsEnabled);
+			try {
+				const { githubToken } = req.session;
+				if (!githubToken) {
+					tracer.trace("github token missing, calling login()");
+					throw "Missing github token";
```

Comment:
Shouldn't it be `throw new Error("Missing GitHub token")`?

Reply:
Doesn't need to in JS.  In this case, we don't need the stacktrace as it's only used for flow control.